### PR TITLE
Created a new folder;Add:New program

### PR DESCRIPTION
**Pull request by Dev-Soumyaranjan**
I created a separate folder for company coding round questions and added a first program inside it which is basically to find a pin on the basis of some condition.

### CODE REVIEW

File: src/Company_Coding_Round_Questions/Find_PIN.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
+package Company_Coding_Round_Questions;
```
Comment (sarthakbal0987) [inline src/Company_Coding_Round_Questions/Find_PIN.java:1]: Please add the problem statement here so that it will anyone can understand the question.

**Review by sarthakbal0987 — CHANGES_REQUESTED** [review]
Code is fine but i have addresed some missing part of the code,Kindly go trough that inorder to merge this pull request

File: src/Company_Coding_Round_Questions/Find_PIN.java (hunk 1 above)
Comment (Dev-Soumyaranjan) [reply src/Company_Coding_Round_Questions/Find_PIN.java:1]: I have added the problem statement in the code

**Review by Asish-dot-Rout — APPROVED** [review]
Great work✨

**Review by sarthakbal0987 — APPROVED** [review]
Fine🎊 and thanks for changing the code accordinly. Now your code is ready to merged

**Pull request merged**